src/rendition-mixin.js

```diff
@@ -30,8 +30,12 @@ const enableFunction = (loader, playlistUri, changePlaylistFn, enable) => {
   if (enable !== currentlyEnabled && !blacklisted) {
     // Ensure the outside world knows about our changes
     changePlaylistFn();
+    if (currentlyEnabled) {
```

Comment:
minor but can we check `if(enable)` and switch the order of the triggers. At first glance I thought these were wrong because `currentlyEnabled` is true but we trigger `disabled`